lib/wlib/WlibMacro.h

```diff
@@ -0,0 +1,18 @@
+/**
```

Comment:
Wlib file has all the necessary macros so it will make more sense to move this macros code inside Wlib so we would have to include only one file